core/jreleaser-model/src/main/java/org/jreleaser/model/AzureArtifacts.java

```diff
@@ -0,0 +1,182 @@
+    }
+    
+    public String getResolvedFeed() {
+        return Env.resolve("AZURE_ARTIFACTS_" + Env.toVar(name) + "_FEED", feed);
+    }
+
+    public String getResolvedPath() {
+        return Env.resolve("AZURE_ARTIFACTS_" + Env.toVar(name) + "_PATH", path);
+    }
+
+    public String getHost() {
+        return host;
+    }
+
+    public void setHost(String host) {
+        this.host = host;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getPersonalAccessToken() {
+        return personalAccessToken;
+    }
+
+    public void setPersonalAccessToken(String personalAccessToken) {
+        this.personalAccessToken = personalAccessToken;
+    }
+
+    public String getProject() {
+        return project;
+    }
+
+    public void setProject(String project) {
+        this.project = project;
+    }
+
+    public String getOrganization() {
+        return organization;
+    }
+
+    public void setOrganization(String organization) {
+        this.organization = organization;
+    }
+
+    public String getFeed() {
+        return feed;
+    }
+
+    public void setFeed(String feed) {
+        this.feed = feed;
+    }
+
+    public String getPath() {
+        return path;
+    }
+
+    public void setPath(String path) {
+        this.path = path;
+    }
+
+    @Override
+    protected void asMap(Map<String, Object> props, boolean full) {
+        props.put("host", getResolvedHost());
+        props.put("username", isNotBlank(getResolvedUsername()) ? HIDE : UNSET);
+        props.put("personalAccessToken", isNotBlank(getResolvedPersonalAccessToken()) ? HIDE : UNSET);
+        props.put("project", getResolvedProject());
+        props.put("organization", getResolvedOrganization());
+        props.put("feed", getResolvedFeed());
+        props.put("path", getResolvedPath());
+    }
+
+    @Override
+    public String getResolvedDownloadUrl(JReleaserContext context, Artifact artifact) {
+        return resolveUrl(context.fullProps(), artifact);
+    }
+    
+    @Override
+    public String getResolvedDownloadUrl(Map<String,Object> props, Artifact artifact) {
+        return resolveUrl(props, artifact);
+    }
+
+    public String getResolvedUploadUrl(JReleaserContext context, Artifact artifact) {
+        return resolveUrl(context.fullProps(), artifact);
+    }
+    
+
+    private String resolveUrl(Map<String,Object> props, Artifact artifact) {
+        Map<String, Object> p = new LinkedHashMap<>(artifactProps(props, artifact));
+        p.put("azuredevopsHost", host);
+        p.put("organization", organization);
+        p.put("project", project);
+        p.put("feed", feed);
+        p.put("path", path);
+
+        String url = "{{azuredevopsHost}}/{{organization}}/{{project}}/_packaging/{{feed}}/maven/v1/{{path}}";
```

Comment:
This URL seems to imply that uploaded artifacts follow the Maven conventions. I'm unfamiliar with conventions for Azure but, the upload capabilities offered by JReleaser should work for any kind of binaries. Perhaps it would be better to define a `url` property instead of `feed` and `path, like it's done for the Artifactory or S3 uploaders.